lib/server/index.js

```diff
@@ -322,6 +322,8 @@ module.exports.start = function () {
 };
 
 module.exports.stop = function () {
+	log.info('Stopping NodeCG');
```

Comment:
Is this log necessary (or related to this PR)?

Reply:
Yeah we can probably get rid of this log line.